bigbluebutton-html5/imports/ui/components/dropdown/list/component.jsx

```diff
@@ -53,37 +52,53 @@ export default class DropdownList extends Component {
     const { dropdownHide } = this.props;
     const { activeItemIndex } = this.state;
 
-    if ([KEY_CODES.SPACE, KEY_CODES.ENTER].includes(event.which)) {
+    let selectableItems = [];
+    for (let i = 0; i < (this._menu.children.length); i++) {
```

Comment:
Switch to a filter.
`let selectableItems = this._menu.children.filter(item => item.hasAttribute('role'))`